File: src/main/java/net/atos/zac/zoeken/model/ZoekObject.java
/*
 * SPDX-FileCopyrightText: 2021 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

package net.atos.zac.zoeken.model;

import net.atos.zac.zoeken.model.index.ZoekObjectType;

public interface ZoekObject {
    ZoekObjectType getType();
}

File: src/main/kotlin/net/atos/zac/zoeken/model/ZoekObject.kt
/*
 * SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.zoeken.model

import net.atos.zac.zoeken.model.index.ZoekObjectType

interface ZoekObject {
    companion object {
        const val IS_TOEGEKEND_FIELD: String = "isToegekend"
    }

    fun getObjectId(): String

    fun getType(): ZoekObjectType
}
